kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerLocator.java

```diff
@@ -15,19 +15,37 @@
 
 package org.kie.server.services.impl;
 
+import org.kie.server.api.KieServerConstants;
+import org.kie.server.services.impl.storage.cloud.KieServerStateCloudRepository;
+
 public class KieServerLocator {
 
-    private static final KieServerImpl INSTANCE = new KieServerImpl();
-    
+    private static final KieServerImpl INSTANCE;
+    private static String strategy = System.getProperty(KieServerConstants.KIE_SERVER_STARTUP_STRATEGY);
+
     static {
+        if (strategy == null || strategy.length() == 0) {
+            strategy = System.getenv("KIE_SERVER_STARTUP_STRATEGY");
```

Comment:
Please move "KIE_SERVER_STARTUP_STRATEGY" constant to KieServerConstants class

Reply:
Please see the comments above with regarding to 'KIE_SERVER_ID'.